node/core/backing/src/lib.rs

```diff
@@ -336,18 +355,51 @@ async fn make_pov_available(
 	Ok(Ok(()))
 }
 
-async fn request_pov_from_distribution(
+async fn request_pov(
 	tx_from: &mut mpsc::Sender<FromJobCommand>,
 	parent: Hash,
 	descriptor: CandidateDescriptor,
+	from_validator: ValidatorIndex,
 ) -> Result<Arc<PoV>, Error> {
-	let (tx, rx) = oneshot::channel();
-
-	tx_from.send(AllMessages::PoVDistribution(
-		PoVDistributionMessage::FetchPoV(parent, descriptor, tx)
-	).into()).await?;
-
-	rx.await.map_err(Error::FetchPoV)
+	let session_index = request_session_index_for_child(parent, tx_from)
+		.await.map_err(Error::FetchPoVUtil)?
+		.await.map_err(Error::FetchPoVCanceled)?
+		.map_err(Error::FetchPoVRuntime)?;
+	let session_info = request_session_info(parent, session_index, tx_from)
+		.await.map_err(Error::FetchPoVUtil)?
+		.await.map_err(Error::FetchPoVCanceled)?
+		.map_err(Error::FetchPoVRuntime)?
+		.ok_or(Error::FetchPoVNoSuchSession)?;
+	let authority_id = session_info.discovery_keys
+		.get(from_validator.0 as usize)
+		.ok_or(Error::FetchPoVInvalidValidatorIndex)?;
+
+	let (req, pending_response) = OutgoingRequest::new(
+		Recipient::Authority(authority_id.clone()),
+		PoVFetchingRequest {
+			relay_parent: parent,
+			descriptor,
+		},
+	);
+	let full_req = Requests::PoVFetching(req);
+
+	tx_from.send(FromJobCommand::SendMessage(AllMessages::NetworkBridge(
+		NetworkBridgeMessage::SendRequests(
+			vec![full_req],
+			IfDisconnected::ImmediateError
+		)
+	))).await?;
+
+	let response = pending_response.await.map_err(Error::FetchPoV)?;
```

Comment:
This fails if the validator is not already connected also. We'd need retrying logic in backing